src/components/Home.js

```diff
@@ -0,0 +1,9 @@
+// import React from "react";
+// import Books from "./Books";
+// const Home = () => {
+//     return (
+//         <Books />
+//     )
+// }
+
+// export default Home;
```

Comment:
- [ ] Kindly avoid leaving commented pieces of code in your codebase. It makes your codebase unclean and is not a good practice :+1: 
- [ ] You can also delete this component if  it's no longer being used in this project :+1: